src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationTracker.State.cs

```diff
@@ -29,8 +29,7 @@ private class State
                     compilationWithoutGeneratedDocuments: null,
                     declarationOnlyCompilation: null,
                     generatedDocuments: ImmutableArray<SourceGeneratedDocumentState>.Empty,
-                    generatedDocumentsAreFinal: false,
-                    unrootedSymbolSet: null);
```

Comment:
i do like how all the constructors of State don't need to figure out what to pass along.